src/dataModels/SummarySubgrid.js

```diff
@@ -28,11 +28,11 @@ SummarySubgrid.prototype = {
     hasOwnData: true, // do not call setData implicitly
 
     getRowCount: function() {
-        return this.behavior.dataModel.getRowCount() && this.getData().length;
+        return this.getData().length;
     },
 
     getData: function() {
-        return this.behavior.dataModel.dataSource.getGrandTotals() || this.data;
+        return this.data;
```

Comment:
As per discussion I will changed to this 
`return this.data.length ? this.data : this.behavior.dataModel.dataSource.getGrandTotals();`